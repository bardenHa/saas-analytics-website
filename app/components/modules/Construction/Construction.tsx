const Construction = () => (
  <div className="flex items-center justify-center bg-red-500 ">
    <h1 className="p-4 text-4xl font-bold text-center text-white lg:text-7xl">
      Site under construction ğŸš§
    </h1>
  </div>
);

export default Construction;
