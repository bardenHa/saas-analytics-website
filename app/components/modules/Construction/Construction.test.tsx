import { render } from '@testing-library/react';
import Construction from './Construction';

describe('Module Construction', () => {
  it('should render component', () => {
    const construction = render(<Construction />);
    expect(construction).toMatchSnapshot();
  });
});
