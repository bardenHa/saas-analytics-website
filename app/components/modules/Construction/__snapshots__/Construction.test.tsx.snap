// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Module Construction should render component 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <div
        class="flex items-center justify-center bg-red-500 "
      >
        <h1
          class="p-4 text-4xl font-bold text-center text-white lg:text-7xl"
        >
          Site under construction ðŸš§
          <button
            class="hidden p-2 px-12 border border-solid rounded-full border-outline md:inline-block hover:bg-primary hover:border-opacity-0 active:bg-primary-hover"
          >
            mole
          </button>
        </h1>
      </div>
    </div>
  </body>,
  "container": <div>
    <div
      class="flex items-center justify-center bg-red-500 "
    >
      <h1
        class="p-4 text-4xl font-bold text-center text-white lg:text-7xl"
      >
        Site under construction ðŸš§
        <button
          class="hidden p-2 px-12 border border-solid rounded-full border-outline md:inline-block hover:bg-primary hover:border-opacity-0 active:bg-primary-hover"
        >
          mole
        </button>
      </h1>
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
